plugin/trino-hive/src/test/java/io/trino/plugin/hive/BaseHiveConnectorTest.java

```diff
@@ -1593,8 +1595,9 @@ private void testReadNoColumns(Session session, HiveStorageFormat storageFormat)
     @Test
     public void createTableWithEveryType()
     {
+        String tableName = "test_types_table_" + randomNameSuffix();
```

Comment:
If you look for `tableName =` there are quite a bit of non-randomized table names in this class.
Is there a reason why this specific table name needs to be randomized?

Reply:
Those two tests had the same table name and it failed in CI for me.